ScubaDivingSE3/src/view/ScubaDivingView.java

```diff
@@ -246,19 +251,26 @@ public void stateChanged(javax.swing.event.ChangeEvent evt) {
     }// </editor-fold>//GEN-END:initComponents
 
     private void ppo2ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_ppo2ActionPerformed
-        //controller.sortListRecords();
+        controller.simpleCalculation.setPPO2(Float.valueOf(ppo2.getSelectedItem().toString()));
     }//GEN-LAST:event_ppo2ActionPerformed
 
     private void sliderDepthStateChanged(javax.swing.event.ChangeEvent evt) {//GEN-FIRST:event_sliderDepthStateChanged
+        controller.simpleCalculation.setDepth(sliderDepth.getValue());
         depth.setText(String.valueOf(sliderDepth.getValue()));
     }//GEN-LAST:event_sliderDepthStateChanged
 
     private void sliderOxygenStateChanged(javax.swing.event.ChangeEvent evt) {//GEN-FIRST:event_sliderOxygenStateChanged
-        Oxygen.setText(String.valueOf(sliderOxygen.getValue()));    }//GEN-LAST:event_sliderOxygenStateChanged
+        controller.simpleCalculation.setOxygen(sliderOxygen.getValue());
+        Oxygen.setText(String.valueOf(sliderOxygen.getValue()));
+    }//GEN-LAST:event_sliderOxygenStateChanged
 
-    private void jButton2ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jButton2ActionPerformed
+    private void eadButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_eadButtonActionPerformed
         // TODO add your handling code here:
-    }//GEN-LAST:event_jButton2ActionPerformed
+    }//GEN-LAST:event_eadButtonActionPerformed
+
+    private void modButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_modButtonActionPerformed
+        simpleResult.setText(String.valueOf(controller.simpleCalculation.modCalculation()));
```

Comment:
Did you test the operation? If values are not set before, result should generate error. That is to say, set values before calculating